hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/storage/row/RowDataParquetWriteSupport.java

```diff
@@ -53,7 +53,7 @@ public WriteContext init(Configuration configuration) {
   @Override
   public void prepareForWrite(RecordConsumer recordConsumer) {
     // should make the utc timestamp configurable
-    this.writer = new ParquetRowDataWriter(recordConsumer, rowType, schema, true);
+    this.writer = new ParquetRowDataWriter(recordConsumer, rowType, schema, false);
```

Comment:
Can we make it configurable?

Reply:
add config "bulk_insert.write.utc-timezone"